README.md

```diff
@@ -49,3 +49,36 @@ Some time later, at 2016-12-29, the customer buys a premium subscription again.
 
 ## Code Improvements
 The function `Ugly_Function` is quite ugly and inefficient, and we want an improved version. Read and understand the function, and implement a more readable and efficient version in `pretty_function`.
+
+## Creating Price Intervals for Product Search
+A feature of our search system, is that customers can choose intervals of prices, e.g.:
+
+* 0-100
```

Comment:
I think technically this is called *-100 (e.g. everything under 100 - not just 0 to 100).
Not sure if it makes it easier or harder to understand though, since negative prices isn't a thing